setup.cfg

```diff
@@ -4,7 +4,7 @@ multi_line_output=3
 include_trailing_comma=True
 known_standard_library=numpy,setuptools
 skip_glob=*/__init__.py
-known_myself=segment_anything
+known_myself=segment_anything_hq
```

Comment:
This bit of configuration isn't crucial - it only relates to [`isort`](https://pycqa.github.io/isort/index.html) configuration, to ensure that when python code imports are sorted, they are sorted in a sensible fashion. Strangely, although `isort` documents several other `known_...` settings (eg [`known_third_party`](https://pycqa.github.io/isort/docs/configuration/options.html#known-third-party)), it _doesn't_ document `known_myself`, and that identifier [doesn't appear anywhere in the `isort` codebase](https://github.com/search?q=repo%3APyCQA%2Fisort+myself&type=code).